superset-frontend/src/explore/components/DataTablesPane/index.tsx

```diff
@@ -255,7 +255,7 @@ export const DataTablesPane = ({
   };
 
   return (
-    <SouthPane>
+    <SouthPane data-test="DataTablesPane">
```

Comment:
Super small nit. I think I have seen a pretty adopted convention for the `data-test` to be hyphen separated or just lower case

Reply:
I agree with you, but I chose to go that way for the same reason here:  https://github.com/apache/superset/pull/13638#discussion_r600734118

Reply:
Ahh, this is a little counter to my comment/thought on that thread. In the linked example, the data-test attr uses the compnent name _inside_ the component (at the outer DOM layer), so any instance of the component will have that data-test attr. I'm fine with that.

In this case, this is a _particular, purposeful instance_ of _a_ SouthPane component. In this case, I think we should follow the convention and go with the `data-test="some-purposeful-instance"` convention.

Reply:
Okay, I don't have such a strong opinion. As they are already "2 against" we will keep it that way :-)